source/testnet/installation/downloads.rst

```diff
@@ -1,94 +1,75 @@
-.. _`Concordium ID APK`: https://client-distribution-testnet.concordium.com/wallet-testnet-release-0.5.30.apk
-.. _TestFlight: https://apps.apple.com/dk/app/testflight/id899247664?l=da
-.. _`TestFlight beta program`: https://testflight.apple.com/join/5LgqqrJ4
 
 .. _downloads:
 
-=========
-Downloads
-=========
+======================
+Installation downloads
+======================
 
 .. contents::
    :local:
    :backlinks: none
 
-.. _concordium_id:
+This topics contains information about where you can download the Concordium Wallets and tools. You can also find out about the hardware requirements for running a node.
 
-Concordium ID
+Mobile Wallet
 =============
 
-Installation on Android
------------------------
+The Concordium Mobile Wallet is available for iOS and Android. The Mobile Wallet supports iOS 13 or later and Android 8 or later.
 
-Installation requirements: Android 8 or newer.
+-  iOS: Go to the App Store and search for Concordium Mobile Wallet.
 
-1. Download the `Concordium ID APK`_ to your phone.
-2. You may see a warning about downloading the .apk-file. If you are asked whether you want to keep the .apk-file, press **OK**.
-3. After opening the .apk-file, you may see a warning saying that your phone is not allowed to install apps from this source. Press **Settings**, toggle **Allow from this source** and go back.
-4. You are now asked if you want to install the application. Press **Install**.
+-  Android: Go to Google Play and search for Concordium Mobile Wallet.
 
+Concordium Desktop Wallet
+=========================
 
-Installation on iOS
--------------------
-
-Installation requirements: iOS 13 or newer.
+-  `Download for Windows <https://distribution.mainnet.concordium.software/tools/windows/concordium-desktop-wallet-1.0.0.exe>`_
 
-Installing the Concordium ID app on iOS requires installation of Apple’s TestFlight app, which is used to distribute beta versions of iOS apps.
+-  `Download for macOS <https://distribution.mainnet.concordium.software/concordium-desktop-wallet-1.0.0.dmg>`_
 
-1. Install `TestFlight`_ via App Store.
-2. Join our `TestFlight beta program`_ via your iPhone (it works best if you click the link directly on your iPhone), and follow the steps shown in there, to add Concordium ID to TestFlight.
-3. Open the TestFlight app on your iPhone and install Concordium ID.
-
-.. _concordium-node-and-client-download:
+-  Download for Linux
 
-.. _concordium-node-and-client:
+   -  `AppImage <https://distribution.mainnet.concordium.software/tools/linux/concordium-desktop-wallet-1.0.0.AppImage>`_
 
-Concordium Node and Client
-==========================
+   -  `Debian package <https://distribution.mainnet.concordium.software/tools/linux/concordium-desktop-wallet-1.0.0.deb>`_
 
-Concordium node and client software for :ref:`Open Testnet v4 <open-testnet-v4>`:
+   -  `RPM <https://distribution.mainnet.concordium.software/tools/linux/concordium-desktop-wallet-1.0.0.rpm>`_
```

Comment:
Perhaps consult with @orhoj about what the minimal requirements for this are (and the same for the .deb package) (i.e., which distributions are supported).

If this is possible to determine.

Reply:
Electron has some information on their site, but we haven't tested on all the platforms they mention. https://www.electronjs.org/docs/tutorial/support